appintro/src/main/java/com/github/paolorotolo/appintro/AppIntroViewPager.kt

```diff
@@ -180,11 +188,10 @@ class AppIntroViewPager(context: Context, attrs: AttributeSet) : ViewPager(conte
     /**
      * Checks if two points are aligned and could represent a slide gesture from the user.
      */
-    private fun isASwipeGesture(startPoint: MotionEvent, x: Float, y: Float) =
-        (
-            (startPoint.x - x).absoluteValue >= VALID_SWIPE_THRESHOLD_PX_X &&
-                (startPoint.y - y).absoluteValue <= VALID_SWIPE_THRESHOLD_PX_Y
-            )
+    private fun isASwipeGesture(startPoint: MotionEvent, x: Float, y: Float) = (
+        isSwipeForward(x, startPoint.x) &&
```

Comment:
Please leave as it was before as the `isASwipeGesture` is just checking if the swipe is horizontal (without direction).